### PR TITLE
Add UI to the deployment

### PR DESCRIPTION
**Pull request by tomusdrw**
This PR adds an additional docker service to the compose file which contains the UI static files hosted by a nginx server.

The image is based on Nikos' work https://github.com/paritytech/parity-bridges-ui/pull/197
Since the image is not being published regularly yet (see https://github.com/paritytech/parity-bridges-ui/issues/201) I'm using the test version.

There is one additional thing that should be addressed (but can be done independently from this PR imho):
- [x] Currently the image has nodes addresses "hardcoded" we need to be able to configure that with `environment` variables so that it can be pointing to local instances instead of the hosted ones. I've added a note about that in [#201](https://github.com/paritytech/parity-bridges-ui/issues/201).

### CODE REVIEW

**Review by svyatonik — APPROVED** [review]
> Currently the image has nodes addresses "hardcoded"

This may be very confusing for those who are running deployments locally :) Hope there aren't many

**Comment by tomusdrw** [comment]
> This may be very confusing for those who are running deployments locally :) Hope there aren't many

Yes. I'm considering changing this PR to a draft until https://github.com/paritytech/parity-bridges-ui/issues/201 is resolved.

File: deployments/ui/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: '3.5'
+services:
+  bridges-ui:
+    # TODO [ToDr] switch to paritytech after https://github.com/paritytech/parity-bridges-ui/issues/201
+    image: wirednkod13/parity-bridges-ui
```
Comment (hbulgarini) [inline deployments/ui/docker-compose.yml:5]:
```suggestion
    image: paritytech/parity-bridges-ui
```

**Review by hbulgarini — COMMENTED** [review]
You can go ahead and replace the UI docker image with the one pushed on `paritytech` hub

**Comment by hbulgarini** [comment]
>  Currently the image has nodes addresses "hardcoded" we need to be able to configure that with environment variables so that it can be pointing to local instances instead of the hosted ones. I've added a note about that in #201.

If you agree with proposed approach on PR https://github.com/paritytech/parity-bridges-ui/pull/273 this can be solved already.

**Comment by tomusdrw** [comment]
> If you agree with proposed approach on PR paritytech/parity-bridges-ui#273 this can be solved already.

This doesn't work unfortunately. The `.env` file affects the image that is being built, but docker compose references an already-built image. We need something that will modify the paths in the runtime (i.e. before starting the container). I have an even more hacky solution somewhere on a branch already, will try to finish this up soonish.

**Review by hbulgarini — APPROVED** [review]
The PR looks good to me even if it still is under draft status.

**Pull request merged**